flight/drone/optical_flow_attribute.py

```diff
@@ -0,0 +1,41 @@
+
+class OpticalFlow(object):
+    """
+    The Optical Flow readings.
+
+    The message definition is here: https://pixhawk.ethz.ch/mavlink/#OPTICAL_FLOW
+
+    :param time_usec: Timestamp
+    :param sensor_id: Sensor ID
+    :param flow_x: Flow in X sensor direction
+    :param flow_y: Flow in Y sensor direction
+    :param flow_comp_m_x: Flow in x-sensor direction, angular-speed compensated
+    :param flow_comp_m_y: Flow in y-sensor direction, angular-speed compensated
+    :param quality: Optical flow quality / confidence. 0: bad, 255: maximum quality
+    :param ground_distance: Ground distance. Positive value: distance known. Negative value: Unknown distance
+    """
+    def __init__(self, time_usec=None, sensor_id=None, flow_x=None,
+            flow_y=None, flow_comp_m_x=None, flow_comp_m_y=None, quality=None,
+            ground_distance=None):
+        """
+        OpticalFlow object constructor.
+        """
+        self.time_usec = time_usec
+        self.sensor_id = sensor_id
+        self.flow_x = flow_x
+        self.flow_y = flow_y
+        self.flow_comp_m_x = quality
+        self.flow_comp_m_y = flow_comp_m_y
+        self.quality = quality
+        self.ground_distance = ground_distance
+
+    def __str__(self):
+        """
+        String representation used to print the OpticalFlow object.
+        """
+        return "OPTICAL_FLOW: time_usec={},sensor_id={},flow_x={},flow_y={}, \
```

Comment:
Consider making the format string a static class constant to reduce the longevity of this line.